packages/slice-machine/tests/server/env.test.js

```diff
@@ -236,76 +231,6 @@ describe("getEnv", () => {
     expect(env.framework).toEqual("vanillajs");
   });
 
-  test("it's client should use autentication token from .prismic", async () => {
```

Comment:
has this changed?

Reply:
In a sense yes, we do get the token and then ask the API.
I didn't wanted to refactor more for now.